### PR TITLE
feat: Add Preact tests

### PR DESCRIPTION
**Pull request by tobySolutions**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [] Code changes have been tested against prettier, or there are no code changes
- [] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [x] This PR changes the codebase
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
  - [ ] This PR changes the internal workings with no modifications to the external API (bug fixes, performance improvements)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: test/preact-server.test.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+import { parseSync } from '@babel/core';
+import million from '../packages/compiler';
+
+
+const BABEL_CONFIG = {
+  plugins: ['@babel/plugin-syntax-jsx', million.babel],
+};
+
+
+describe('react-compiler', () => {
```
Comment (aidenybai) [inline test/preact-server.test.tsx:11]: preact-compiler

File: test/preact.test.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { describe, expect, it } from 'vitest';
+import { block as createBlock } from '../packages/million';
```
Comment (aidenybai) [inline test/preact.test.tsx:4]: remove this file

**Review by aidenybai — CHANGES_REQUESTED** [review]

File: test/preact.test.tsx (hunk 2 above)
Comment (tobySolutions) [reply test/preact.test.tsx]: Done!

File: test/preact-server.test.tsx (hunk 1 above)
Comment (tobySolutions) [reply test/preact-server.test.tsx]: Done!

**Comment by tobySolutions** [comment]
You can check now @aidenybai. Let me know if there is anything else to fix.

**Review by aidenybai — APPROVED** [review]

**Pull request merged**